R/cut.R

```diff
@@ -113,6 +113,13 @@ prep.step_cut <- function(x, training, info = NULL, ...) {
   col_names <- recipes_eval_select(x$terms, training, info)
   check_type(training[, col_names], types = c("double", "integer"))
 
+  if (!is.numeric(x$breaks)) {
+    cli::cli_abort(
+      "{.arg breaks} must be a numeric vector, \\
```

Comment:
Do we still need \\ in errors and warnings? I think that the cli functions remove linebreaks

Reply:
And the reason this isn't done even though these are being added here is because they're technically just relocated from elsewhere in the repo?

Reply:
yes that is correct